.eslintrc

```diff
@@ -6,15 +6,26 @@
         "import",
         "promise"
     ],
+    "overrides": [
+        {
+            "files": ["src/*.js"],
+            "settings": {
+                "flowtype": {
+                    "onlyFilesWithFlowAnnotation": false
+                }
+            }
+        },
+        {
+            "files": ["**/*_test.js"],
+            "rules": {
+                "max-lines": "off"
+            }
+        }
+    ],
     "rules": {
-        "flowtype/require-exact-type": [
```

Comment:
We're using exact by default and this rule doesn't understand that the syntax is therefore ok.